src/vaults/Vault.ts

```diff
@@ -135,21 +163,28 @@ export class Vault {
     }
 
     authorized_for_role(public_key: string, role: string) {
-        /* "owners" role is authorized for everything */
-        if (this.meta.roles.owners && this.meta.roles.owners[public_key]) return true;
+        /* OWNERS_ROLE role is authorized for everything */
+        if (this.meta.roles[Vault.OWNERS_ROLE] && this.meta.roles[Vault.OWNERS_ROLE][public_key]) return true;
 
         return !!(this.meta.roles[role] && this.meta.roles[role][public_key]);
     }
 
-    authorized_role(public_key: string) {
-        /* return "owners" first if we're an owner" */
-        if (this.meta.roles.owners && this.meta.roles.owners[public_key]) return 'owners';
+    authorized_roles(public_key: string) {
+        const roles = [];
+
+        /* return OWNERS_ROLE first if we're an owner" */
+        if (this.meta.roles[Vault.OWNERS_ROLE] && this.meta.roles[Vault.OWNERS_ROLE][public_key]) {
+            roles.push(Vault.OWNERS_ROLE);
+        }
 
         /* or return the first role we are authorized for... */
```

Comment:
This change makes it so now you are returning a list of authorized roles - not sure if this is intentional or if the comment is just outdated

Reply:
This was intentional.  Uses of `authorized_role` were updated to `authorized_roles` and now handle multiple return values.  Comment will be updated.